py/occampy3/wrappers/sbm_manager.py

```diff
@@ -16,3 +16,18 @@ def __init__(self, ref=None):
 
     def make_model(self, model_name: str, make_project: bool) -> Model:
         return Model(self._ref.makeSbModel(model_name, make_project))
+
+    def getTestSampleSz(self) -> double:

```

Comment:
I think you have the function names mixed up here. The pyoccam wrapper function is `getTestSampleSz`, and the C++ function is `getTestSampleSize`.